app/controllers/catalog_controller.rb

```diff
@@ -57,17 +57,29 @@ def self.modified_field
 
     # solr fields that will be treated as facets by the blacklight application
     #   The ordering of the field names is the order of the display
-    config.add_facet_field solr_name("human_readable_type", :facetable), label: "Type", limit: 5
-    config.add_facet_field solr_name("resource_type", :facetable), label: "Resource Type", limit: 5
+    # config.add_facet_field solr_name("human_readable_type", :facetable), label: "Type", limit: 5  # TODO: issue#133:
+    # config.add_facet_field solr_name("resource_type", :facetable), label: "Resource Type", limit: 5 # TODO: issue#133:
+    # config.add_facet_field solr_name("creator", :facetable), limit: 5
+    # config.add_facet_field solr_name("contributor", :facetable), label: "Contributor", limit: 5
+    # config.add_facet_field solr_name("keyword", :facetable), limit: 5 # TODO: issue#133:
+    # config.add_facet_field solr_name("subject", :facetable), limit: 5 # TODO: issue#133:
+    # config.add_facet_field solr_name("language", :facetable), limit: 5
+    # config.add_facet_field solr_name("based_near_label", :facetable), limit: 5  # TODO: issue#133:
+    # config.add_facet_field solr_name("publisher", :facetable), limit: 5 # TODO: issue#133:
+    # config.add_facet_field solr_name("file_format", :facetable), limit: 5
+    # config.add_facet_field solr_name('member_of_collections', :symbol), limit: 5, label: 'Collections'
+
+    # TODO: issue#133: skip these fields: unit, external group, date digitized
+    config.add_facet_field solr_name("date_issued", :facetable), label: "Publication Date", limit: 5
+    config.add_facet_field solr_name("date_created", :facetable), limit: 5
+    config.add_facet_field solr_name("date_uploaded", :facetable), limit: 5 # aka date_ingested
     config.add_facet_field solr_name("creator", :facetable), limit: 5
     config.add_facet_field solr_name("contributor", :facetable), label: "Contributor", limit: 5
-    config.add_facet_field solr_name("keyword", :facetable), limit: 5
-    config.add_facet_field solr_name("subject", :facetable), limit: 5
     config.add_facet_field solr_name("language", :facetable), limit: 5
-    config.add_facet_field solr_name("based_near_label", :facetable), limit: 5
-    config.add_facet_field solr_name("publisher", :facetable), limit: 5
+    config.add_facet_field solr_name("genre", :facetable), label: "Genre", limit: 5, helper_method: :capitalize_value
     config.add_facet_field solr_name("file_format", :facetable), limit: 5
     config.add_facet_field solr_name('member_of_collections', :symbol), limit: 5, label: 'Collections'
+    config.add_facet_field solr_name('workflow_state_name', :symbol), limit: 5, label: 'Workflow State', helper_method: :capitalize_value
```

Comment:
This works for now, but I think we'll want to copy over the functionality of Avalon 6.x of showing/hiding this facet based upon authorization.  Probably the same for date_uploaded and possibly others.